scala/scala.bzl

```diff
@@ -595,8 +595,8 @@ def _collect_jars_from_common_ctx(ctx, extra_deps = [], extra_runtime_deps = [])
 
     return struct(compile_jars = cjars, transitive_runtime_jars = transitive_rjars, jars2labels=jars2labels, transitive_compile_jars = transitive_compile_jars)
 
-def _format_full_jars_for_intellij_plugin(full_jars):
-    return [struct (class_jar = jar, ijar = None) for jar in full_jars]
+def _format_full_jars_for_intellij_plugin(full_jars, ijar):
+    return [struct (class_jar = jar, ijar = ijar) for jar in full_jars]
```

Comment:
@chaoren what does the plugin do with the `ijar` field?
I set it to None since I didn't see any drawbacks from setting it to None and on the other hand it becomes a bit more tricky to match the java-code ijar to the java-code full jar and the scala-code ijar to the scala-code full jar. It's not rocket science but a bit more cumbersome than I want and without any value I can see.

@greggdonovan the above implementation might be a problem since it passes the scala-code ijar for the java-code full jar. Depending on @chaoren's response we'll need to:
1. Change the intellij tests to expect None (if he agrees we don't need it).
2. Change our code to pass the right tuples so that we pass the right ijar to the full jar (if he surfaces reasons to using the ijar).

Reply:
btw, if @chaoren says this can be ignored we can merge this PR and later remove it once he fixes the test

Reply:
Not needed. But we'll need to change the intellij aspect to accommodate the divergence from java.

Reply:
It's possible to output multiple jars now?

Reply:
We actually just use the `class_jar` if the ijar is not available, so you might see some performance hits.

Reply:
What performance hit? In loading bigger jars to memory?
@greggdonovan this sounds like the current change pleases the test but will break the actual production

Reply:
Yeah pretty much. class_jar can always replace ijar, except it's much larger.

Reply:
I see. 
@greggdonovan do you want to take a crack at passing the matching ijar to the relevant full jar?